src/com/gmail/St3venAU/plugins/ArmorStandTools/Main.java

```diff
@@ -184,7 +184,7 @@ void setPlayerSkull(Player p, ArmorStand as) {
             p.sendMessage(ChatColor.RED + Config.noAirError);
             return;
         }
-        b.setType(Material.SIGN);
+        b.setType(Material.OAK_WALL_SIGN);
```

Comment:
Shouldn't this be a Sign post?

Reply:
https://github.com/St3venAU/ArmorStandTools/pull/31#discussion_r422658297